src/test/java/fixture/PositionFixture.java

```diff
@@ -0,0 +1,33 @@
+package fixture;
+
+import domain.position.File;
+import domain.position.Position;
+import domain.position.Rank;
+
+public final class PositionFixture {
+    public static Position createC1() {
+        return new Position(new File('c'), new Rank(1));
+    }
```

Comment:
메서드로 제공하는 대신 객체를 미리 생성해 두는 것은 어떤가요?
```suggestion
    public static final Position C1 = new Position(new File('c'), new Rank(1));
```